metricbeat/beater/runner.go

```diff
@@ -0,0 +1,59 @@
+package beater
+
+import (
+	"sync"
+
+	"github.com/elastic/beats/libbeat/publisher"
+)
+
+// ModuleRunner is a facade for a ModuleWrapper that provides a simple interface
```

Comment:
I would prefer to have all the Wrapper and Runner stuff in a separate package, but didn't come up with a good name yet.